### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by koachcode**
Update the game over the message so people know how to play again!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (koachcode) [inline index.html]: - This is a Good Message, as it tells the user what to do after the game ends. Good job

**Review by koachcode — COMMENTED** [review]
Good Work

File: index.html (hunk 1 above)
Comment (Ginohmk) [inline index.html:50]: - I suggest you can make the test more exciting by adding exclamation and more emojis

Comment (koachcode) [inline index.html]: - I suggest you can make the test more exciting by adding exclamation and more emojis 
```suggestion
<h2 hidden>Game over!!!, refresh to play again 🧑‍💻 🤖!</h2>
```

**Pull request merged**